src/data/staticContentJsons/2021-MASTER.json

```diff
@@ -117,6 +117,7 @@
           "units": "Household",
           "topic_code": "DEM",
           "base_url_2021": "https://ons-dp-prod-census-maps-dem-mig.s3.eu-west-2.amazonaws.com",
+          "base_url_2011_2021_comparison": "https://ons-dp-prod-census-maps-comparison-2011.s3.eu-west-2.amazonaws.com",
```

Comment:
All of these `base_url_2011_2021_comparison` values will need to come out before we can merge to develop, I think